### PR TITLE
Update changelog/package.json for 3.14.0 release

### PR DESCRIPTION
**Pull request by Blackbaud-SteveBrush**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+# 3.14.0 (2020-03-30)
+
+- Added the `enablePointerEvents` option to `SkyOverlayConfig` which enables click events to pass through the overlay's backdrop. [#148](https://github.com/blackbaud/skyux-core/pull/148)
```
Comment (blackbaud-johnly) [inline CHANGELOG.md]: Seems like `which enables` should be `to enable`. I mean, it's the option that enables this, not the configuration, right?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,10 @@
...
+- Added the `backdropClick` event to the `SkyOverlayInstance` to allow overlay instances to listen for backdrop click events. [#148](https://github.com/blackbaud/skyux-core/pull/148)
```
Comment (blackbaud-johnly) [inline CHANGELOG.md]: I'd either delete `the` before `SkyOverlayInstance`. (Alternately, we could add a noun after `SkyOverlayInstance`, but I'm not sure what that noun would be. Is this a component?)

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,10 @@
...
+- Added the `isTargetAboveElement()` method to the `SkyCoreAdapterService` to allow SKY UX components to detect if an event target element is contained within another element. [#148](https://github.com/blackbaud/skyux-core/pull/148)
```
Comment (blackbaud-johnly) [inline CHANGELOG.md]: I'd delete `the` before `SkyCoreAdapterService`. (Alternately, we could add `service` as a noun after `SkyOverlayInstance`, but that seems redundant.)

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,10 @@
...
+- Fixed the `closed` event on the `SkyOverlayInstance` to emit when the instance is closed by either the `SkyOverlayService` or the instance itself. [#148](https://github.com/blackbaud/skyux-core/pull/148)
```
Comment (blackbaud-johnly) [inline CHANGELOG.md]: I'd either delete `the` before both instances of `SkyOverlayInstance`. (Alternately, we could add a noun after `SkyOverlayInstance`, but I'm not sure what that noun would be. Is this a component?)

Also, it seems like `emit` should either be `fire` here or it should be followed by whatever it emits. ... I'd also delete `either`, but that might just be me.

File: CHANGELOG.md (hunk 2 above)
Comment (Blackbaud-SteveBrush) [reply CHANGELOG.md]: Removed "the" throughout. The `SkyOverlayInstance` is a simple class, not a component.

**Pull request merged**